src/components/ApartmentApplicantList/index.js

```diff
@@ -71,6 +99,33 @@ export default class ApplicantList extends Component {
                 )}
               </List>
             </Grid>
+            <Grid item xs={12} sm={6}>
+              {this.props.applicants.length < MIN_NUM_APPLICANTS && this.state.saveButtonClicked ? (
+                <Typography variant="overline" color="error">
+                  Must have at least {MIN_NUM_APPLICANTS} applicants
+                </Typography>
+              ) : null}
+            </Grid>
+            <Grid item xs={12} sm={3}>
+              {this.props.saving ? (
+                this.props.saving === 'success' ? (
```

Comment:
Should you use 'savingSuccess' here and on 113 instead of saving? You would have to rearrange the if-else pattern here, if so (by making the GordonLoader case the 'else' of the outer-if and not the second inner-if).

Reply:
Hmm, on second thought, then the loader would not show in the case of "it is saving but it is undecided about failure or success"..

Reply:
I thought I had completely removed "savingSuccess" and was using "saving" as both an indicator of saving in progress and indicator of saving complete, making the "savingSuccess" variable uneccessary. (Saving = false means not in progress, so show the save button. True means saving, so show progress circle. "Success" means saving it done and successful, so it will register like saving=true to hide the button, but the checking the value will tell it to give the checkmark instead of progress wheel. And "failed" is same idea but with error icon)
Since that savingSuccess variable is still there, then it either got messed up when I did a merge or maybe I forgot to commit/push a change from one of my computers

Reply:
I seems I made that change 2 branches up from this branch, so that is my fault for not thinking to include those things with the "save-application" branch.